pyproject.toml

```diff
@@ -2,6 +2,7 @@
 requires = [
     "setuptools>=43",
     "wheel",
+    "Cython==3.0.10;implementation_name == 'graalpy'",
```

Comment:
Does it need to be graalpy specific? The official cython recommendation is not to include generated sources in source distributions anymore (https://github.com/cython/cython/pull/6201), so I'd do it for all pythons and frame it as general improvement towards best practices.

Reply:
Good idea